gammapy/cube/tests/test_psf_kernel.py

```diff
@@ -0,0 +1,62 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import pytest
+import numpy as np
+from numpy.testing import assert_allclose
+import astropy.units as u
+from astropy.coordinates import Angle
+from ...maps import WcsNDMap, MapAxis, WcsGeom
+from .. import PSFKernel, table_psf_to_kernel_map
+from ...irf import TablePSF, EnergyDependentTablePSF
+
+
+def test_table_psf_to_kernel_map():
+    sigma = 0.5 * u.deg
+    binsz = 0.1 * u.deg
+    geom = WcsGeom.create(binsz=binsz, npix=150)
+
+    rad = Angle(np.linspace(0., 3 * sigma, 100), 'deg')
+    table_psf = TablePSF.from_shape(shape='gauss', width=sigma, rad=rad)
+    kernel = table_psf_to_kernel_map(table_psf, geom, normalize=True)
+
+    # Is normalization OK?
+    assert_allclose(kernel.data.sum(), 1.0, atol=1e-5)
+
+    # maximum at the center of map?
+    ind = np.unravel_index(np.argmax(kernel.data, axis=None), kernel.data.shape)
+    # absolute tolerance at 0.5 because of even number of pixel here
+    assert_allclose(ind, geom.center_pix, atol=0.5)
```

Comment:
Those two asserts are pretty indirect. Suggest this instead:
```
assert kernel.data.shape == (some, value)
assert kernel.data[center, pixel] == kernel.data.max()
```
I.e. put the shape values and center pixel index values, isntead of this atol=0.5

Just computing a kernel that's 3 x 3 and putting the 3x3 float values in the assert might also be an option.